src/components/AppNavigation/Settings/SettingsAddressbook.vue

```diff
@@ -27,7 +27,7 @@
 			<div class="addressbook__content">
 				<!-- addressbook name -->
 				<span class="addressbook__name" :title="addressbook.displayName">
-					{{ addressbook.displayName }}
+					{{ addressbook.displayName }} {{ !addressbook.enabled ? ' ' + t('contacts', '(Disabled)') : '' }}
```

Comment:
Not sure if this is the correct way and works with all languages. We might as well use an l10n string `{addressbookname} (Disabled)`

Reply:
Can you add a code suggestion?

Reply:
```suggestion
					{{ addressbook.enabled ? {{ addressbook.displayName }} : t('contacts', '{addressbookname} (Disabled)', addressbook.displayName) }}
```

Reply:
Thanks! :)

Reply:
Tested and works! :)